ui/src/app/pages/login/account-creation/account-creation.tsx

```diff
@@ -131,9 +131,9 @@ export class AccountCreation extends React.Component<AccountCreationProps, Accou
         },
         institutionalAffiliations: [
           {
-            institution: undefined,
+            institution: '',
             nonAcademicAffiliation: undefined,
```

Comment:
All other fields in this object are either strings or boolean values... is there a reason why we believe 'undefined' is safe for the nonAcademicAffiliation field? If so, we should add a comment explaining why it's different from all the other fields.

Reply:
nonAcademicAffiliation is ENUM field while institution is string since it is an existing field i can add comment